src/snapshot_bundle.cc

```diff
@@ -24,6 +25,7 @@ namespace toit {
 
 static const char* const MAGIC_NAME = "toit";
 static const char* const MAGIC_CONTENT = "like a tiger";
+static const char* const SHA_NAME = "sha256";
```

Comment:
Let's turn this into uuid id, so all other tools can use the same and it fits into Program::Header.